corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -110,11 +110,13 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
         var changeFormLanguage = toggles.toggleEnabled('CHANGE_FORM_LANGUAGE');
         var enablePrintOption = !menuResponse.queryKey;
         var sidebarEnabled = toggles.toggleEnabled('SPLIT_SCREEN_CASE_SEARCH') && !appPreview;
+        var smallScreenEnabled = window.innerWidth <= constants.SMALL_SCREEN_WIDTH_PX;
 
         if (sidebarEnabled && menuResponse.type === "query") {
             var menuData = menusUtils.getMenuData(menuResponse);
             menuData["triggerEmptyCaseList"] = true;
             menuData["sidebarEnabled"] = true;
+            menuData["smallScreenEnabled"] = smallScreenEnabled;
```

Comment:
wouldn't this exclude the case when sidebar is enabled and the request type is entities? The would enter the second if statement here

Reply:
That's true. I was looking at where `sidebarEnabled` is set here, but I see that case is handled differently. Does that mean it wants to have its initial value set in `getMenuView` instead? https://github.com/dimagi/commcare-hq/blob/1c0353a5eae1f900b3472e6936b0155009759f33/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/utils.js#L163-L165 Or in addition to?

Or maybe it'd be better defined in the view itself if it needs to be there no matter how the view is called (per your last comment that small screen mode should work with all features on their own).

Reply:
right the sidebarEnabled is (luckily?) a bit more annoying-- I think getMenuView or within CaseListView would both work as long as it's also available to CaseTileView (which is should since CaseTileView extends CaseView with is a child of CaseListView)

Reply:
I realized I could use the listener util to set the initial value as well, so I went that route instead, with the listener called in `CaseListView` (and inherited by `CaseTileListView`). I'm pretty sure we're going to need the `smallScreenEnabled` value in `QueryListView` at some point too, so this way it'll be easier to add it there when/if we do.